src/libraries/Microsoft.Extensions.DependencyInjection.Specification.Tests/src/Microsoft.Extensions.DependencyInjection.Specification.Tests.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <EnableDefaultItems>true</EnableDefaultItems>
+    <CLSCompliant>false</CLSCompliant>
+    <IsPackable>true</IsPackable>
+    <TargetsForTfmSpecificBuildOutput>$(TargetsForTfmSpecificBuildOutput);IncludeProjectReferencesInPackage</TargetsForTfmSpecificBuildOutput>
```

Comment:
nit: Consider encapsulating the P2P as private asset functionality as a target that can be enabled via an msbuild property.

Reply:
I thought about this, but expect that @Anipik will probably have some more generic implementation based on other package requirements (EG: packaging content from other projects in different paths).  I suspect analyzers/source-generators will hit this.

Reply:
I'm fine with that but let's make sure that this specific code path then gets replaced by the more generic implementation. Unsure how to best track this.

Reply:
@Anipik did you have a general-purpose pattern in mind for CSProj's that want to include the output of other projects?

Reply:
Do we really want to do include `Microsoft.Extensions.DependencyInjection.Abstractions` in this package?

Reply:
DI.Abstractions is added as a package dependency and not inlined into the package as its ProjectReference doesn't set `Pack=true`.